flux/flux-core/src/test/resources/configs/config-methods-test.yaml

```diff
@@ -70,8 +70,17 @@ bolts:
             - "foo"
             - "bar"
             - "baz"
+  - id: "bolt-2"
+    className: "org.apache.storm.flux.test.TestBolt"
+    factory: "newInstance"
+    parallelism: 1
 
-
+  - id: "bolt-3"
+    className: "org.apache.storm.flux.test.TestBolt"
+    factory: "newInstance"
+#    factoryArgs:
```

Comment:
What if we don't comment this two lines? If it just works, I think this two lines should not be commented out. If it doesn't work, we may need to make it work, cause it is not actual tested.